druid/src/contexts.rs

```diff
@@ -186,6 +186,25 @@ impl_context_method!(
             self.widget_state.size()
         }
 
+        /// The origin of the widget in window coordinates.
+        pub fn window_origin(&self) -> Point {
+            self.widget_state.window_origin()
+        }
+
+        /// Takes a point in widget coordinates and transforms it to
+        /// window coordinates (relative to the content area, excluding chrome)
+        pub fn to_window(&self, widget_point: Point) -> Point {
+            self.window_origin() + widget_point.to_vec2()
+        }
+
+        /// Takes a point in widget coordinates and transforms it to
+        /// screen coordinates
```

Comment:
```suggestion
        /// Convert a point from the widget's coordinate space to the screen's.
```

Can we add a link to the screen module here? I've never really looked at that so not sure what would be best to link.